did/plugins/phabricator.py

```diff
@@ -197,51 +245,324 @@ class Differential:  # pylint: disable=too-few-public-methods
 
     """
 
-    def __init__(self, data, verbose: bool = False):
-        if verbose:
-            self.str = f'{data["fields"]["uri"]} {data["fields"]["title"]}'
-        else:
-            self.str = f'D{data["id"]} {data["fields"]["title"]}'
+    def __init__(self, data):
+        self._phid = data["phid"]
+        self._uri = data["fields"]["uri"]
+        self._title = data["fields"]["title"]
+        self._id = data["id"]
+
+    @property
+    def phid(self) -> str:
+        """
+        Returns the Phabricator ID for the differential as a string
+        """
+        return self._phid
+
+    @property
+    def uri(self) -> str:
+        """
+        Returns the Phabricator URI for the differential as a string
+        """
+        return self._uri
+
+    @property
+    def title(self) -> str:
+        """
+        Returns the Phabricator title for the differential as a string
+        """
+        return self._title
+
+    @property
+    def id(self) -> str:
+        """
+        Returns the Phabricator ID for the differential as a string
+        """
+        return self._id
+
+    def __str__(self):
+        """ String representation """
+        if DifferentialsBaseStats.verbose:
+            return f'{self.uri} {self.title}'
+        return f'D{self.id} {self.title}'
```

Comment:
```suggestion
        return f'D{self.id} - {self.title}'
```

Let's make the default format consistent with other stats.